src/utilities/multicall.ts

```diff
@@ -0,0 +1,60 @@
+import { Fragment, Interface, JsonFragment } from '@ethersproject/abi';
+import { CallOverrides, Contract } from '@ethersproject/contracts';
+import { JsonRpcProvider } from '@ethersproject/providers';
+
+import { MultiCallAddress } from '../config';
+import { SupportedChainId } from '../types';
+
+export interface Call {
+  address: string; // Address of the contract
+  name: string; // Function name on the contract (example: balanceOf)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  params?: any[]; // Function params
```

Comment:
Use `unknown` here instead of `any` since it provides better type safety.

Technically `unknown` is the proper type here since the `params` for a `Call` are in fact, an unknown type.

`unknown` = "What this is, is unknown to me, and me knowing what it is will not be important for my usage."
`any` = "This could be anything, and I need to be able to interact with it. I will most likely need to figure out some information about what it is at a later time."

TypeScript has matured a lot and there's very few cases where you actually need to use `any`.